apps/molgenis-components/src/components/tables/TableSimple.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="table-responsive">
+    <table class="table table-bordered" :class="{ 'table-hover': tableHover }">
+      <thead>
+        <tr>
+          <th scope="col" style="width: 1px" v-if="hasColheader">
+            <slot name="colheader" />
+          </th>
+          <th v-for="col in columns" :key="col" scope="col">
+            {{ col }}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="row in rows" :key="JSON.stringify(row)">
+          <td v-if="hasColheader">
+            <div style="display: flex">
+              <slot name="rowheader" :row="row" />
+              <input
+                class="form-check form-check-inline mr-1"
+                v-if="selectColumn"
+                type="checkbox"
+                :checked="isSelected(row)"
+                @click="onRowClick(row)"
+              />
+            </div>
+          </td>
+          <td
+            v-for="col in columns"
+            :key="col"
+            @click="onRowClick(row)"
+            style="cursor: pointer"
+          >
+            <ul v-if="Array.isArray(row[col])" class="list-unstyled">
+              <li v-for="(item, index3) in row[col]" :key="index3">
+                {{ item }}
+              </li>
+            </ul>
+            <span v-else-if="row[col]">{{ flattenObject(row[col]) }}</span>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+/** Data table. Has also option to have row selection. Selection events must be handled outside this view. */
+export default {
+  props: {
+    /** the column names */
+    columns: Array,
+    /** list of rows matching column names */
+    rows: Array,
+    /** set to create select boxes that will yield this columns value when selected. */
+    selectColumn: String,
+    /** default value */
+    defaultValue: Array,
+  },
+  data: function () {
+    return {
+      selectedItems: [],
+    };
+  },
+  watch: {
+    selectedItems() {
+      this.$emit("input", this.selectedItems);
+    },
+  },
+  created() {
+    if (this.defaultValue instanceof Array) {
+      this.selectedItems = this.defaultValue;
+    } else {
+      this.selectedItems.push(this.defaultValue);
+    }
+  },
```

Comment:
This function should not be necessary, since defaultValue is defined as an array in props (line 58). Alternatively the definition is incorrect.